### DISCUSSION TITLE
How to simulate disable soft constraints?

### DISCUSSION THREAD

**Discussion by eliran-arbili** [fvutils/pyvsc#218] [Q&A]
Hi,
Is there any way to disable all soft constraints following the rules determined by the SystemVerilog IEEE?

![image](https://github.com/fvutils/pyvsc/assets/5417496/307df782-8a84-4005-a4bd-c35f4e62680e)

**Comment by mballance** [comment]
PyVSC doesn't currently support 'disable soft'. Let me give some consideration to how difficult it would be to support it.
